src/game/state.py

```diff
@@ -62,3 +65,41 @@ def check_position_availability(self, x, y):
 
     def switch_player(self):
         self.game_turn.player_one_active = not self.game_turn.player_one_active
+
+    def check_winner(self):
+        if self.check_raws_winning() or self.check_columns_winning() or self.check_diags_winning():
+            if self.game_turn.player_one_active:
+                player_name = "Player 1"
+            else:
```

Comment:
Useless else

Reply:
If I remove the "else", I will have a useless double allocation 1 time /2 when setting the "player 1" name
I think I perfer this righting for now